src/packagedcode/gemfile_lock.py

```diff
@@ -26,8 +26,10 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import attr
```

Comment:
This should come after standard imports

Reply:
This is no use now because i have problem that class gem is of unhashable type

Reply:
https://github.com/nexB/scancode-toolkit/pull/1694/files#diff-ebf0b433a58c067b70ad8235d3e08c5eR210 . This line gives `TypeError: unhashable type: 'Gem' `.So how to use attrs here hor hashing.

Reply:
My point was only about the ordering of imports `This should come after standard imports`... this is not a standard library import and we sort imports in this order: standard, third-party then scancode-own modules.

Reply:
Ok, first solve that error , then think about ordering . Because if you find other way , then we have to delete this import.